code/API_definitions/geofencing.yaml

```diff
@@ -901,3 +938,48 @@ components:
           device:
             phoneNumber: +123456789
           terminationReason: SUBSCRIPTION_EXPIRED
+
+    REQUEST_POLYGON_AREA_ENTERED:
+      value:
+        webhook:
+          notificationUrl: https://application-server.com
+          notificationAuthToken: c8974e592c2fa383d4a3960714
+        subscriptionDetail:
+          device:
+            phoneNumber: "+12345678912"
+          area:
+            areaType: POLYGON
+            boundary:
+              - latitude: 52.516770
+                longitude: 13.378156
+              - latitude: 52.516841
+                longitude: 13.379433
+              - latitude: 52.515917
+                longitude: 13.37959
+              - latitude: 52.515849
+                longitude: 13.378308
+          type: org.camaraproject.geofencing.v0.area-entered
```

Comment:
Please bear in mind we have another PR https://github.com/camaraproject/DeviceLocation/pull/191 to rename security scopes and even types, so you may need to adapt. Thank you.

Reply:
I recommend not updating the security scope and event types here until the #191 PR is merged. This can be done afterwards through merging and resolving any merge conflicts.

Otherwise, we will have "features" or "updates" that are not part of this PR/issue resolution. Additionally, if I update #191, we would also need to update it here.

Reply:
This feature is not in scope for next meta-release, so for now we keep this in Draft mode.